pombola/south_africa/templates/core/organisation_people.html

```diff
@@ -9,6 +9,14 @@
 {% block content %}
 
   <h2>People at {{ object.name }}</h2>
+
+  <div class="people-list-inline-search">
+    <span class="">Search for people:</span>
+    <form class="inline-search" action="{% url "core_search" %}">
+      <input class="search-autocomplete-name" id="core_search" data-source="/search/autocomplete/?model=person" type="text" name="q" value="" placeholder="Name"></input>
```

Comment:
Do you think it's worth having a hidden input here which restricts the `core_search` results to people?

```html
<input type="hidden" name="section" value="persons">
```